### ISSUE TITLE
How do I include a .well-known directory in the dist directory (e.g. Let's Encrypt)?

### ISSUE THREAD

**Issue by plainspace** [11ty/eleventy#1049]

**Comment by Ryuno-Ki** [comment]
Mind sharing your solution, @plainspace? :-)

**Comment by pdehaan** [comment]
@plainspace You might need to manually copy the folder: https://www.11ty.dev/docs/copy/

**Comment by plainspace** [comment]
So... I had to do the following to make this work:

1. I created a directory in `/src/site/.well-known/` with 2 relevant `filename.njk` files. Each had the following frontmatter:
```
---
permalink: /.well-known/filename
eleventyExcludeFromCollections: true
---
{
 json stuff
}
```

When the site builds, the files get created in `/dist/.well-known/filename`

2. Because I'm deploying on Netlify I had to add the following to my `netlify.toml`
```
[[headers]]
  for = "/.well-known/*"
  [headers.values]
    content-type = "application/json"
```

I'm going to have to file a bug because this isn't working as expected.
